rust_src/src/frames.rs

```diff
@@ -279,4 +285,17 @@ pub fn frame_position(frame: LispObject) -> LispObject {
     }
 }
 
+/// Returns t if the mouse pointer displayed on FRAME is visible.
+/// Otherwise it returns nil. FRAME omitted or nil means the selected frame.
+/// This is useful when `make-pointer-invisible` is set
+#[lisp_fn(min = "0")]
+pub fn frame_pointer_visible_p(frame: LispObject) -> LispObject {
+    let frame_ref = frame_or_selected(frame);
+    if frame_ref.pointer_invisible() {
```

Comment:
You can actually return the bool directly. Just return `!frame_Ref.pointer_invisible()` directly and update the function definition to reflect `bool` instead of `LispObject`.

I really need to update the docs. Thanks for the reminder.

Reply:
Fixed.